src/js/routes/Ballot/Ballot.jsx

```diff
@@ -129,7 +129,7 @@ export default class Ballot extends Component {
       </div> :
       <div></div>;
 
-    const ballotItems = ballot !== null ?
+    const ballotItems = typeof ballot !== "undefined" && ballot !== null ?
```

Comment:
Checking for just "ballot" seems to handle both "ballot !== null" and "ballot !== undefined" situations.